benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/tasks.py

```diff
@@ -89,6 +99,28 @@ def generate_request(model_params, prompt):
         raise ValueError(f"Unknown backend: {backend}")
     return pload
 
+def get_token_count(prompt, resp):
+    """Get number of tokens to prompt and resp using the tokenizer"""
+    global tokenizer
+    backend = model_params["backend"]
+    
+    number_of_input_tokens = len(tokenizer.encode(prompt))
+    number_of_output_tokens = 0
+    
+    if backend == "vllm":
+        number_of_output_tokens = 0 # to be added
```

Comment:
What is the difference in output tokens across different model server token backends? Are the outputs not in the resp.content here for other model server backends or is this not in utf-8 format or something else?

Reply:
+1

Reply:
The specific API response body is different for different model servers. vLLM, TGI and TensorRT-LLM all are different when it comes to the actual API response. So, we need to handle them individually.

Reply:
I have an implementation for tgi in a WIP PR. I think it's okay for us to implement these other frameworks as a follow up task.